conn_request.go

```diff
@@ -354,7 +395,10 @@ func (req *connRequest) Accept() (Conn, error) {
 	}
 
 	// Create a new socket ID
-	socketId := uint32(time.Since(req.ln.start).Microseconds())
+	socketId, err := req.generateSocketId()
+	if err != nil {
+		return nil, fmt.Errorf("could not generate socket id")
```

Comment:
Nit (I don't think this is code you added either)
```suggestion
		return nil, fmt.Errorf("could not generate socket id: %w", err)
```

Reply:
Done